.github/workflows/ci.yml

```diff
@@ -12,27 +12,27 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Check out code
-        uses: actions/checkout@master
+        uses: actions/checkout@main
       - name: Run Chef Delivery
-        uses: actionshub/chef-delivery@master
+        uses: actionshub/chef-delivery@1.0.0
```

Comment:
Let's please leave all of these to either `main` (if it exists) or `master`.

Reply:
IMHO, attaching to master leaves the outcome of the tests in a potentially unpredictable state, as the HEAD of master/main could be potentially unstable. I'd rather leave it at a specific version that has been fully tested and released knowing that the code can be stable.

Reply:
While I agree, we also manage most of these actions and know the state of the main/master branches. We'd like to be able to push any updates that are needed easily to cookbooks instead of having to bump them. Our standard has been to use the branches and not tags on all the other cookbooks.

Reply:
Then why have releases at all?